File: Karan-Example-Project-Solutions/Python/Numerical/Fibonacci.py
def fibonacci(n):
    if (n <= 1):
        return n
    return fibonacci(n-1) + fibonacci(n-2)

user = input("Enter number of terms: ")
n = int(user)
print("The Fibonacci Sequence up to", n, "terms is:")
for i in range(0, n):
    print(fibonacci(i), end=" ")

File: Karan-Example-Project-Solutions/Java/Numerical/Fibonacci.java
import java.util.Scanner;

public class Fibonacci {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    System.out.println("Enter number of terms: ");
    int n = input.nextInt();
    System.out.println("Fibonacci Sequence up to " + n + " terms:");

    System.out.print(0);
    for (int i = 1; i <= n; i++) {
      System.out.print(", " + fibonacci(i));
    }
    System.out.println();
  }

  private static int fibonacci(int n) {
    if (n <= 1) {
      return n;
    }
    return fibonacci(n-1) + fibonacci(n-2);
  }
}
